superset/sql_parse.py

```diff
@@ -187,6 +187,14 @@ def __eq__(self, __o: object) -> bool:
 
 
 class ParsedQuery:
+    class TableTuple(NamedTuple):
+        table: Table
+        level: int
+
+    class AliasTuple(NamedTuple):
+        name: str
+        level: int
```

Comment:
Nit: while I'm having a hard time coming up with a better name for these, I'm not a big fan of Hungarian notation. An alternative could be `TableWithLevel` and `AliasWithLevel` to clarify the contents rather than the type.

Reply:
As a fellow hater of Hungarian notation, I'd like to stress I'm not the original author :D I'll change these to something nicer.